ant/easy/filter.py

```diff
@@ -47,25 +47,30 @@ def wait_for_message(match, process, queue, condition):
                 queue.remove(message)
                 condition.release()
                 return process(message)
-            elif message[1] == 1 and message[2][0] in [
-                Message.Code.EVENT_TRANSFER_TX_FAILED,
-                Message.Code.EVENT_RX_FAIL_GO_TO_SEARCH,
-            ]:
+            # SDS BUG : need to check for actual channel number, but it's not passed in
+            # SDS : original code temp disabled
+            """
+            elif message[1] == 1 and message[2][0] in [Message.Code.EVENT_TRANSFER_TX_FAILED,
+                                                       Message.Code.EVENT_RX_FAIL_GO_TO_SEARCH]:
                 _logger.warning("Transfer send failed:")
                 _logger.warning(message)
                 queue.remove(message)
                 condition.release()
                 raise TransferFailedException()
+            """
```

Comment:
Is there a solution to this?

Reply:
not in my latest commit, I'm afraid. I've only highlighted that there is an issue with the filter.py code, because it should also match the event with the right channel. For instance RX_FAIL_GO_TO_SEARCH could be a misconfigured channel from another device.